app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StaticPagesController < ApplicationController
+  skip_before_action :require_login, :only => [:home]
+  def home
+    @user = User.new
+    redirect_to user_path(current_user.id) if signed_in_user?
+  end
+
+  def timeline
+    @profile = false
```

Comment:
Are you still using your static pages routes? Might want to pull them out into something RESTful if so